index.js

```diff
@@ -117,6 +117,10 @@ class ServerlessWebpack {
       'webpack:serve:serve': () => BbPromise.bind(this)
         .then(this.validate)
         .then(this.serve),
+
+      'before:offline:start': () => BbPromise.bind(this)
+        .then(this.validate)
+        .then(this.compile),
```

Comment:
just trowing this our there, would a "watch" here instead of a "compile" make sense?